.github/workflows/c8run-linux-x86-build.yaml

```diff
@@ -0,0 +1,73 @@
+name: "test c8run on linux"
+
+on:
+  push:
+    paths:
+      - "c8run/**"
+      - ".github/workflows/c8run-linux-x86-build.yaml"
+
+jobs:
+  test_c8run:
+    name: Test deploying camunda platform on c8run
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: disable and stop mono-xsp4.service
+        run: |
+             sudo systemctl stop mono-xsp4.service || true
+             sudo systemctl disable mono-xsp4.service || true
+             sudo killall mono || true
+             sudo killall xsp4 || true
+
+      - uses: actions/checkout@v4
+
+      - name: print architecture
+        run: arch
+
+      - name: make a package
+        run: ./package.sh
+        working-directory: ./c8run
+
+      - name: Run c8run
+        run: nohup ./start.sh &
+        working-directory: ./c8run
+        env:
+          JAVA_HOME: /usr/lib/jvm/temurin-21-jdk-amd64
+
+      - name: Sleep 2 mins
+        run: sleep 120
```

Comment:
Is there any check/probe that can be used instead of a sleep here (not a good practice) to make the test more robust?

Reply:
I think c8run now supports the probes out-of-the-box, so that sleep shouldn't be necessary anymore. I will remove them and address CI issues as needed.

Reply:
I've removed the sleeps.